src/pages/case-study/learnerbly-accelerating-professional-growth.js

```diff
@@ -169,31 +166,34 @@ const IndexPage = props => {
           <BlockRow
             mobile={{ bottom: '5', top: '4' }}
             tablet={{ bottom: '6', top: '6' }}
+            style={{ justifyContent: 'flex-end' }}
           >
-            <Col width={[1, 1, 1, 1, 1 / 2, 4 / 12]}>
-              <ReactMarkdown
-                renderers={{
-                  // eslint-disable-next-line
-                  heading: props => <SectionTitle {...props} />,
-                  // eslint-disable-next-line
-                  paragraph: props => <StyledBodyPrimary {...props} />
-                }}
-                source={normalise(data7).text}
-              />
+            <Col width={[1, 1, 1, 1, 1, 4 / 12]}>
+              <Row>
+                <Col width={[1, 1 / 2, 1 / 2, 1 / 2, 1 / 2, 1]}>
```

Comment:
Ok so wtf is going on here?

Previously we were hiding BodyPrimary on large then showing on smaller screens. Whilst not a bad solution we do have the tools available that means we don't have to have duplicate content. 

So we've pretty much made something that looks like this:
![image](https://user-images.githubusercontent.com/7108120/61429497-25085f80-a959-11e9-8873-62fdd7695959.png)

In the image, `[ ]`  represents a `<Col />`. So you can see we've nested like this:
```
<Col>
    <Row>
        <Col>
           Header 
        </Col>
        <Col>
            BodyPrimary
        </Col>
    </Row>
</Col>
<Col>
    <Image />
</Col>
```

Reply:
Nesting is wicked for this! I didn't think of it but great stuff 👍 I wished I did think of it.